board/fischertechnik/TXT/linux_defconfig

```diff
@@ -7,6 +7,7 @@ CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=16
 CONFIG_BLK_DEV_INITRD=y
+CONFIG_INITRAMFS_SOURCE="$${BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH}/output/initramfs/initramfs.cpio"
```

Comment:
so here is the dependency of the second build from the first?

Reply:
yes. We cache the initramfs.cpio in this directory during build process